scripts/check_names.py

```diff
@@ -704,7 +704,9 @@ def comprehensive_parse(self):
             "include/mbedtls/*.h",
             "drivers/builtin/include/mbedtls/*.h",
             "drivers/everest/include/everest/everest.h",
-            "drivers/everest/include/everest/x25519.h"
+            "drivers/everest/include/everest/x25519.h",
+            "drivers/everest/include/tf-psa-crypto/private/everest/everest.h",
+            "drivers/everest/include/tf-psa-crypto/private/everest/x25519.h"
```

Comment:
Non-blocker: we should put a comma after each list item, even the last one. This would avoid having to change the last element when adding another item after it. We usually do that, this file was one of the minority that doesn't.